<launch>
  <arg name="robot_name"/>
  <rosparam file="$(find utm_traffic)/map/voronoi_utm_demo8.yaml" command="load"/>
  <rosparam file="$(find utm_traffic)/map/voronoi_utm_demo8_cells.yaml" command="load"/>
  <node pkg="utm_traffic" type="pioneer_traffic" respawn="false" name="pioneer_traffic" output="screen">
    <param name="robot_name" value="'$(arg robot_name)'"/>
    <remap from="membership" to="$(arg robot_name)/membership"/>
  </node>
</launch>
